.gitignore

```diff
@@ -0,0 +1,2 @@
+.project
+.settings
```

Comment:
w/ the slash, it'll only recognize a .settings dir (not a file). Not a big deal as a .settings file is probably not likely -